metadata/src/test/java/org/apache/kafka/image/node/ClusterImageBrokersNodeTest.java

```diff
@@ -66,7 +66,7 @@ public void testNode1Child() {
             "rack=Optional.empty, " +
             "fenced=false, " +
             "inControlledShutdown=false, " +
-            "isMigratingZkBroker=false)", child.stringify());
+            "isMigratingZkBroker=false, directories=[])", child.stringify());
```

Comment:
let's have some output in this test, not just an empty directory, to verify that this all works